pages.nl/linux/apt-get.md

```diff
@@ -0,0 +1,37 @@
+# apt-get
+
+> Hulpprogramma voor pakketbeheer van Debian en Ubuntu.
+> Zoek naar pakketten met `apt-cache`.
+> Meer informatie: <https://manpages.debian.org/latest/apt/apt-get.8.html>.
+
+- Werk de lijst van beschikbare pakketten en versies bij (het wordt aanbevolen dit uit te voeren voor elk ander `apt-get` commando):
+
+`apt-get update`
+
+- Installeer of werk het pakket bij naar de laatste beschikbare versie:
+
+`apt-get install {{package}}`
```

Comment:
```suggestion
- Installeer specifieke pakketten of werk ze bij naar de nieuwste beschikbare versies:

`apt-get install {{pakket1 pakket2 ...}}`
```

Optional fix (error source: original page): broken description: plural noun expected.
Mandatory fix (error source: original page): [broken placeholder](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#special-cases): translation with ellipsis expected.
Note: translation source: Install specific packages, or update them to the latest available versions.

Reply:
Thanks for the review! Fixed in the latest commit.